products/appengine/inspec.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Inspec::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  FirewallRule: !ruby/object:Overrides::Inspec::ResourceOverride
+    exclude: true
+  StandardAppVersion: !ruby/object:Overrides::Inspec::ResourceOverride
+    exclude: false
+    properties:
+      id: !ruby/object:Overrides::Inspec::PropertyOverride
+        override_name: "version_id"
```

Comment:
Instead of using `override_name`, I think we want something like this:
https://github.com/GoogleCloudPlatform/magic-modules/blob/d39a680a8a989b2547f5e55426dd619dc8ac8d9c/products/appengine/terraform.yaml#L45

That will rename it both for the singular and plural resource

Reply:
Renamed it.

Reply:
Looks like "service" and "env_variable" are coming back as nil.  As of now I am going to remove "service" property validation from test. Will check with Ali, if he has more info.

Reply:
Yeah, I believe they are coming back as `nil`, along with `libraries` and maybe a couple others. I would prefer if we could mark these properties as `exclude` in `inspec.yaml` so that we won't generate docs for them. 

It would be a bad user experience to see documentation on a bunch of fields that won't work!

Reply:
I have added these. Hope this will be my last change.. 👍 Tested locally. Need to keep same name of properties similar to api.yaml

![image](https://user-images.githubusercontent.com/17652733/64453918-ac4f9500-d0b7-11e9-8d13-7b9181bc1be9.png)